client/src/constants.rs

```diff
@@ -0,0 +1,12 @@
+/// The default `keep_alive` interval for a client connection.
+pub const KEEP_ALIVE_DEFAULT: u64 = 5_000;
+
+#[cfg(debug_assertions)]
+pub(crate) const CLOUD_CA: &[u8; 455] = include_bytes!("../cloud.debug.der");
+#[cfg(debug_assertions)]
+pub(crate) const SELIUM_CLOUD_REMOTE_URL: &str = "127.0.0.1:4001";
```

Comment:
Should we stick with 7001 for consistency?

Reply:
I'll set it to 7002 just so that there's no collisions with the dev server 👍